### ISSUE TITLE
Compatibility with Nuclide

### ISSUE THREAD

**Issue by tonyxiao** [steelbrain/flow-ide#17]
Can I use this together with Nuclide? (for other nuclide functionalities).

**Comment by steelbrain** [comment]
Nuclide has an option that you can toggle to enable support for linter providers, try enabling it and tell us if it works

**Comment by tonyxiao** [comment]
Oh I use linter package instead of nuclide's diagnostics (which does not seem to support linting current editor buffer). So nuclide diagnostics is actually disabled. However I was wondering if I need to disable nuclide's other flow support feature, such as autocompletion.

**Comment by steelbrain** [comment]
Yeah you probably should disable Nuclide's flow autocompletion because this package also includes the autocompletion support
